xds/googledirectpath/googlec2p.go

```diff
@@ -192,7 +192,7 @@ func newXdsServerConfig(xdsServerURI string) string {
 	{
 		"server_uri": "%s",
 		"channel_creds": [{"type": "google_default"}],
-		"server_features": ["xds_v3", "ignore_resource_deletion", "xds.config.resource-in-sotw"]
+		"server_features": ["ignore_resource_deletion"]
```

Comment:
why did we get rid of the `sotw` server feature? mm.. maybe that's the only thing we support right?